packages/release-mastermind/src/contexts/methods/checkStale.ts

```diff
@@ -0,0 +1,154 @@
+import { LoggingLevels } from "@videndum/utilities";
```

Comment:
Codacy has a fix for the issue: [There should be no space after '{'.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
import {LoggingLevels } from "@videndum/utilities";
```